changelogs/unreleased/gh-7581-downstream-lag-calculation-fix.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/replication
+
+* Fixed `box.info.replication[id].downstream.lag` growing indefinitely on a
+  server when it's not authoring any new transactions (gh-7581).
```

Comment:
I would change "authoring" to "sending" or "transmitting".

Reply:
Changed to "writing". Seems more clear IMO.